components/Header/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import { SITE_TITLE } from '../../constants/main';
+import { MenuItem } from '../common/MenuItem';
+import { SecondaryButton } from '../common/Button';
+import { Wrapper, Logo, MenuItems, UserWrapper } from './styles';
+import { LoginButton } from '../common/LoginButton';
+import Link from 'next/link';
+import { Icon, IconSizes, IconTypes } from '../common/Icon';
+import { useMediaQuery } from '../../hooks/useMediaQuery';
+import { device } from '../../styles/mediaQueries';
+
+const Header = () => {
+  const router = useRouter();
+  const isTablet = useMediaQuery(device.tablet);
+  const [isTabletMenuDisplayed, setIsTabletMenuDisplayed] = useState(false);
+
+  const handleMenuIconClick = () => {
+    console.log('clicked');
+    setIsTabletMenuDisplayed(!isTabletMenuDisplayed);
+  };
+
+  return (
+    <header>
+      <Wrapper>
+        <div className="top-row">
+          <Link href={'/'}>
+            <a>
+              <Logo>
```

Comment:
There is the Logo component.

Reply:
added <Logo/> component
We need to investigate a warning in the console. 
Warning: Prop 'fill' did not match. Server: '...' Client: '....'
Seems to be related to changing Logo colors.